keyboards/keychron/k8/rgb/info.json

```diff
@@ -0,0 +1,33 @@
+{
+    "keyboard_name": "Keychron K8",
+    "manufacturer": "Keychron",
+    "url": "https://www.keychron.com/products/keychron-k4-wireless-mechanical-keyboard",
+    "processor": "SN32F248BF",
+    "maintainer": "tbowmo",
+    "bootloader": "sn32-dfu",
+    "debounce": 1,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "rgb_matrix": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "dip_switch_enable": true
+    },
+    "rgb_matrix": {
+        "driver": "SN32F24xB"
```

Comment:
does this work?

Reply:
Yup, it does show up in the autogenerated rules.mk file:
`RGB_MATRIX_ENABLE ?= yes`

Reply:
somehow it doesn't get picked up here. weird

Reply:
obj_keychron_k8_rgb_iso/src/info_rules.mk:RGB_MATRIX_ENABLE ?= yes